### DISCUSSION TITLE
Ports are not accessible in truenas scale chia app

### DISCUSSION THREAD

**Discussion by MarcoBonino** [Chia-Network/chia-blockchain#18004] [Support]
Hello!

I’m using TrueNAS-SCALE-23.10.2.

The Chia app seems not allowing connections to it from the exposed ports. As per [Chia | TrueNAS Documentation Hub](https://www.truenas.com/docs/scale/scaletutorials/apps/communityapps/chia/) I setup the full node port to 38444 (also tried with a different number). The app is configured as full node and it runs fine.
The problem is when I try to connect to it. I’m not able to connect to the full node even from the local network and of course I tried from a remote node (all needed ports are open on the router).
I found this since I was not able to connect using a remote harvester (on port 38447). It seems both ports are not reaching the app.

Do you have an idea on how to ''open" these ports ?

Thanks!

**Comment by BrandtH22** [comment]
Hey @MarcoBonino , the TrueNAS integration has been created and is managed by community members. 
You would need to contact the community members who are managing that in order to have any updates or changes implemented.

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
